src/main/java/pl/nogacz/snake/application/Achievements.java

```diff
@@ -0,0 +1,227 @@
+package pl.nogacz.snake.application;
+
+
+import java.io.*;
+import java.util.Scanner;
+import java.awt.*;
+import java.awt.TrayIcon.MessageType;
+public class Achievements {
+
+    private int[] achieved;
+   private double[] progress;
+    Design design;
+
+    public Achievements(Design design){         // Constructor loads achievement status from src/main/AchievementData folder
+        this.design=design;
```

Comment:
fix spacing. Tüm kod değişikliklerinde bunu uygula. Birçok yerde problem var. Tek tek hepsini etiketleyemiyorum.

`this.design = design`